src/atoms/Size/__tests__/Size.web.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+import View from '../../View';
+import Size from '../index';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
```

Comment:
Why do we need this?

Reply:
@kamleshchandnani we need this to suppress errors logged by React component into console while testing for "error" cases:

<img width="520" alt="Screenshot 2020-04-02 at 6 51 34 PM" src="https://user-images.githubusercontent.com/9150371/78254091-06f88480-7513-11ea-8339-08631956180c.png">

Reply:
that happens for tests that are testing error cases. So this suppress should be part of that particular test and not `beforeAll` which runs before each test run.